src/reducers/clusterReducer.js

```diff
@@ -179,6 +179,24 @@ export default function clusterReducer(
         items: items,
       };
 
+    case types.CLUSTER_LOAD_STATUS_SUCCESS:
+      items = Object.assign({}, state.items);
+
+      items[action.clusterId] = Object.assign({}, items[action.clusterId], {
+        status: action.status,
+      });
+
+      return {
+        lastUpdated: state.lastUpdated,
+        isFetching: false,
+        items: items,
+      };
+
+    case types.CLUSTER_LOAD_STATUS_ERROR:
+      return {
+        numberOfNodes: 0,
```

Comment:
Should this be something different? I don't think my code per se uses this, so what would be correct here?

Reply:
```suggestion

        lastUpdated: state.lastUpdated,
        isFetching: false,
        items: items,
```

Indeed I would return the state without changes. Same as we do with other errors

Reply:
Roger. I checked example from load cluster details error and it's same here now.